File: 3396-valid-word/3396-valid-word.cs
public class Solution {
    public bool isVowel(char ch){
        ch = char.ToLower(ch);
        return "aeiou".Contains(ch);
    }
    public bool IsValid(string word) {
        if(word.Length < 3)return false;
        bool v = false,c = false;
        foreach(char ch in word){
            if(!char.IsLetterOrDigit(ch))return false;
            if(char.IsLetter(ch)){
                if(isVowel(ch))v=true;
                else c = true;
            }
        }
        return (v&&c);
    }
}

File: 3396-valid-word/3396-valid-word.cpp
class Solution {
public:
    bool isVowel(char c) {
        c = tolower(c); 
        return c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u';
    }
    bool isValid(string word) {
        if(word.size() < 3)return false;
        bool v = false,c = false;
        for(auto &ch : word){
            if(!isalpha(ch) && !isdigit(ch))return false;
            if(isVowel(ch))v=true;
            else if(isalpha(ch)) c=true;
        }
        return (v&&c);
    }
};